### DISCUSSION TITLE
2024-02-05/

### DISCUSSION THREAD

**Discussion by giscus** [justwink/justwink.github.io#60] [Q&A]
# 2024-02-05/

There are two colors in my head.

https://iakoug.cn/2024-02-05/

**Comment by koteramomo** [comment]
青山精神病院的郝院长曾经说过，过多的情绪反应容易降低寿命。因此当人们在发觉情绪问题时，应避免大量冲动的情绪反应，及时采取灵活摆烂的处事手段。（灵活摆烂即：在社会环境中行事时，根据事情大小灵活选择摆烂或保持理性，避免冲动的情绪化反应造成后续社死，然后在远离社会环境后及时发疯、及时暴饮暴食、及时摆烂、及时睡大觉，高举尊重但不放任的伟大旗帜，全面贯彻平衡并灵活调控情绪的中心思想，构建个人特色的理性摆烂的身心保障体系，扎实落实先分场合再谈举措的务实精神，推进压力释放的重要工作。

**Reply by iakoug** [reply]
贯彻落实科学发展观，坚持可持续发展
